journal/original/11-chapter-10.adoc

```diff
@@ -452,6 +453,7 @@ she was again drawn forth to address some individuals in a very particular manne
 it was a time of sweet refreshment in which most present were tendered,
 and I hope the sense of heavenly regard which then
 prevailed will not soon be forgotten by some of us.`"
+//does this above need to be embedded? It seems it was a letter that was written.
```

Comment:
Yes, I think let's embed it as a `.letter` without any salutation or signature, just basically the opening and closing of the embed.